collectors/property.yaml

```diff
@@ -22,8 +22,24 @@ HostSystemPropertiesCollector:
       - label: "runtime_memory_capacity_kilobytes"
         property: "runtime|memoryCap"
 
+VMPropertiesCollector:
+    enum_metrics:
+      - label: "runtime_powerState"
+        expected: "Powered On"
+        property: "summary|runtime|powerState"
+      - label: "runtime_connectionState"
+        expected: "connected"
+        property: "summary|runtime|connectionState"
+      - label: "summary_numEthernetCards"
+        expected: "1"
+        property: "summary|config|numEthernetCards"
 
+    info_metrics:
+      - label: "summary_parentVcenter"
+        property: "summary|parentVcenter"
```

Comment:
You don't have to query this information again. It is already there. Add the vcenter name in `get_vms()` in `InventoryBuilder`. Then add the new label for `vcenter` : `self.vms[vm_id]['vcenter']`

`'vcenter': vcenter.name,`


```
    def get_vms(self):
        tree = dict()
        for vcenter in self.vcenter_list:
            for dc in vcenter.datacenter:
                for cluster in dc.clusters:
                    for host in cluster.hosts:
                        for vm in host.vms:
                            tree[vm.uuid] = {
                                    'uuid': vm.uuid,
                                    'name': vm.name,
                                    'parent_host_uuid': host.uuid,
                                    'parent_host_name': host.name,
                                    'cluster': cluster.name,
                                    'datacenter': dc.name,
                                    'vcenter': vcenter.name,
                                    'target': vm.target,
                                    'token': vm.token,
                                    }
        self.vms = tree
        return tree
```

Reply:
Yes, i will remove this metrics, I had to put it into collector because of reason i explained - https://github.com/sapcc/vrops-exporter/pull/47#discussion_r396401228